### DISCUSSION TITLE
How to clear Selection from TableBlock?

### DISCUSSION THREAD

**Discussion by Doctor-N0** [Budibase/budibase#12111] [Help] [answered]
That might be a basic question, but how do I clear the selected rows from a Table Block? I'm using the [Bulk Update](https://github.com/Budibase/budibase/discussions/11055) on the Table Block and after hitting the Update Button, I want that no rows are selected. Using the "Clear Form" or "Refresh Data Provider" option on the Button didn't seem to work for me.

Here's my layout:

![Screenshot 2023-10-19 at 11-16-18 Budibase](https://github.com/Budibase/budibase/assets/105864309/aea287b7-08a8-4a00-821c-030a85125f4e)

Thanks so much

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Doctor-N0,

I don't believe theres an action that can achieve this. You can instead just click the very first selectable checkbox at the top left of the table twice to select all then de-select all.

My example in that discussion is a bit deceptive as the gif just loops, giving it the illusion that its de-selected everything.

A better representation of what actually happens would be the example done by Andrew: [linked](https://github.com/Budibase/budibase/discussions/11055#discussioncomment-6370952). You can see that the select stay even after the records have been updated. 

Feel free to create a feature request [here](https://github.com/Budibase/budibase/issues/new/choose) if you wish for an action to be added to the core product to work in the way you've described.

I hope this helps and makes sense.
